src/terrafort/templates/aws_security_group_rule.tf

```diff
@@ -1,25 +1,23 @@
-{% for range in resource.IpRanges %}
+{% for range in resource.IpRanges -%}
 {% set rule_num = count() %}
 resource "aws_security_group_rule" "{{ resource.name }}{% if rule_num > 0 %}_{{ rule_num }}{% endif %}" {
-  security_group_id = "{{ resource.id }}"
+  security_group_id = {{ resource.id }}
   type              = "{{ resource.type }}"
   from_port         = {{ resource.FromPort | default('0')}}
   to_port           = {{ resource.ToPort | default('0')}}
   protocol          = "{{ resource.IpProtocol}}"
   cidr_blocks       = ["{{ range.CidrIp }}"]
   description       = "{{ range.Description }}"
 }
-{% endfor %}
-
-{% for source in resource.UserIdGroupPairs %}
+{% endfor -%}
+{%- for source in resource.UserIdGroupPairs %}
```

Comment:
no newline between these two?

Reply:
no, but the output will still show the newline